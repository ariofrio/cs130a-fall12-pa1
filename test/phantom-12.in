insert 12
insert 56
print

insert 899
insert -50
insert 0
insert -1
insert 1
print

lookup 12
lookup -50
deleteMax
print

deleteMax -50
print 

lookup 56
lookup 12
delete 12
print

lookup 12
deleteMax
print

