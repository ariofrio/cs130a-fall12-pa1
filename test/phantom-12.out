item successfully inserted
item successfully inserted
12 56
56 12
item successfully inserted
item successfully inserted
item successfully inserted
item successfully inserted
item successfully inserted
0 1 12 56 -50 899 -1
899 12 56 -50 0 -1 1
item found
item found
max item 899 deleted
0 1 12 56 -50 -1
56 12 1 -50 0 -1
max item 56 deleted
0 1 12 -50 -1
12 -1 1 -50 0
item not found
item found
item successfully deleted
0 1 -50 -1
1 -1 0 -50
item not found
max item 1 deleted
0 -50 -1
0 -1 -50
